internal/pkg/agent/cmd/upgrade.go

```diff
@@ -53,20 +58,73 @@ func newUpgradeCommandWithArgs(_ []string, streams *cli.IOStreams) *cobra.Comman
 	cmd.Flags().String(flagPGPBytes, "", "PGP to use for package verification")
 	cmd.Flags().String(flagPGPBytesURI, "", "Path to a web location containing PGP to use for package verification")
 	cmd.Flags().String(flagPGPBytesPath, "", "Path to a file containing PGP to use for package verification")
+	cmd.Flags().BoolP(flagForce, "", false, "Advanced option to force an upgrade on a fleet managed agent")
+	err := cmd.Flags().MarkHidden(flagForce)
```

Comment:
@cmacknz Could you please confirm whether we keep this flag hidden or whether to remove the hidden attribute and document it?

Reply:
Keep it hidden, it should not be the primary way to upgrade agents that are Fleet managed, and it doesn't work today regardless.